web/src/routes/StyleGuide.jsx

```diff
@@ -59,6 +65,26 @@ export default function StyleGuide() {
         </Button>
       </div>
 
+      <Heading size="md">Dialog</Heading>
+      <Button
+        onClick={() => {
```

Comment:
lets at least keep it consistent. move this function up near the handle dismiss.